front/app/components/IdeaCards/IdeasList.tsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import styled, { useTheme } from 'styled-components';
+import {
+  Box,
+  media,
+  Button,
+  useWindowSize,
```

Comment:
Why not use `useBreakpoint` instead? It's truly better for performance and it requires less code to implement.